packages/cyberstorm/src/components/Forms/CreateTeamForm/CreateTeamForm.module.css

```diff
@@ -0,0 +1,37 @@
+.createTeamDialog {
```

Comment:
Since we're at least currently using CSS modules, prefixing everything with `createTeam` is unnecessary.